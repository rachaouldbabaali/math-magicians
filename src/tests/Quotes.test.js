import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Quotes from '../components/quotes';

describe('Quotes', () => {
  it('renders the component', () => {
    const tree = renderer.create(<Quotes />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders the loading state', () => {
    render(<Quotes isLoading />);
    const loading = screen.getByText('Loading...');
    expect(loading).toBeInTheDocument();
  });
});
