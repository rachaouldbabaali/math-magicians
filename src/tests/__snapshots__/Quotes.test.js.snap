// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes renders the component 1`] = `
<div
  className="quotes"
>
  Loading...
</div>
`;
